src/routes/Routes.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Route, Switch, BrowserRouter } from 'react-router-dom';
 import HomePage from '../components/home/Home';
-import Painel from '../components/layouts/Painel';
```

Comment:
Bom dia. 

Por que o componente painel tem que ficar dentro da pasta Routes? Se seguir este padrão todo o componente  que for utilizado em uma rota tem que estar na pasta Routes?

Reply:
Oi Dani, 
na vdd, pensando em clean code, pensei em contextualizar os arquivos por quem está usando os mesmos. Qdo o Painel for usado por outros componentes a gente cria uma pasta generica  para ter essa reutilização.